cli/command/cli.go

```diff
@@ -54,7 +55,8 @@ type DockerCli struct {
 	err            io.Writer
 	client         client.APIClient
 	defaultVersion string
-	server         ServerInfo
+	serverInfo     ServerInfo
+	clientInfo     ClientInfo
```

Comment:
We could move `defaultVersion` to `clientInfo` I think, but doesn't need to happen in this PR.